data/json/snippets/snippets.json

```diff
@@ -5,75 +5,75 @@
     "text": [
       {
         "id": "flier_1",
```

Comment:
Just FYI, these ids are saved to the save files, so items from old saves with these ids will have different snippets once loaded with the new version. Not a huge problem in my opinion, but worth noting in case you want to avoid that.

Reply:
That's entirely intentional.